clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -1710,7 +1710,7 @@ class AnnotatingParser {
             TT_UnionLBrace, TT_RequiresClause,
             TT_RequiresClauseInARequiresExpression, TT_RequiresExpression,
             TT_RequiresExpressionLParen, TT_RequiresExpressionLBrace,
-            TT_BracedListLBrace)) {
+            TT_BracedListLBrace, TT_TableGenMultiLineString)) {
```

Comment:
It would be better if you could set the type as finalized.

Reply:
Modified to finalize on finding the token.